ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -42,6 +42,7 @@ module.exports =  Marionette.ItemView.extend({
         initialize: function() {
             this.checkIfDefaultSearchForm();
             this.listenTo(user.getQuerySettings(), 'change', this.checkIfDefaultSearchForm);
+            this.isSystemTemplate();
```

Comment:
You'll want to do this in the onRender method, otherwise the class might get dropped.  We should probably add `checkIfDefaultSearchForm` there as well.